packages/clerk-js/src/ui/contexts/components/Plans.tsx

```diff
@@ -72,7 +72,7 @@ export const PlansContextProvider = ({ children }: PropsWithChildren) => {
     <PlansContext.Provider
       value={{
         componentName: 'Plans',
-        plans: plans || [],
+        plans: plans && subscriptions ? plans : [],
```

Comment:
This breaks the E2E tests, when the user is signed out and still not sure why it was added

Reply:
This has been fixed now